### DISCUSSION TITLE
Isn't this a issue ?

### DISCUSSION THREAD

**Discussion by sumansahoo1** [tailwindlabs/tailwindcss#12267] [General]
![image](https://github.com/tailwindlabs/tailwindcss/assets/75962762/2fdde9df-dc00-422e-82fb-ad26fb5ebca9)
Shouldn't the css file in the step 3 and step 5 be same ?

**Comment by wongjn** [comment]
No, for "generic" PostCSS setup with no other build tool, you'd want the **compiled** CSS file linked in the HTML file. This would be the file that contains the Tailwind-generated rules.

**Reply by wongjn** [reply]
> What did the 4th step does actually ?[^1]

Compiles the CSS rules, using `src/input.css` as the input CSS file and outputting the content to `dist/output.css`, and then watches for further changes.

> Are these the corresponding css code of the tailwind properties ?

Yes.

> If then how it's get updated when I update the code ?

The `--watch` flag included in the command means that the process stays open and will recompile when Tailwind detects a change in relevant files.

> Do I need to compile it everytime?

Yes, though `--watch` flag included in the command means that the process stays open and will recompile when Tailwind detects a change in relevant files, thus recompiling for you.

[^1]: https://github.com/tailwindlabs/tailwindcss/discussions/12267#discussioncomment-7358621

**Reply by sumansahoo1** [reply]
Okay

**Reply by sumansahoo1** [reply]
I did understood how it's working. 

I am seeing this error - The stylesheet http://localhost:3000/dist/output.css was not loaded because its MIME type, “text/html”, is not “text/css”.

And what I understood what this error says is, it's thinking that /dist/output.css is a html file not css. Can you help with this ? 

The discussion is over though. Thanks.

**Comment by sumansahoo1** [comment]
What did the 4th step does actually ? 

because, I didn't did the 4th step but still I can apply tailwind properties. 
I now did and I can see much css in the output.css file. 
Are these the corresponding css code of the tailwind properties ? 

If then how it's get updated when I update the code ? Do I need to compile it everytime?

**Reply by wongjn** [reply]
Replied at https://github.com/tailwindlabs/tailwindcss/discussions/12267#discussioncomment-7361009 to keep the reply chain together.
